src/inmanta/env.py

```diff
@@ -631,8 +667,8 @@ def _write_sitecustomize_py_file(self) -> None:
         Write a sitecustomize.py file to the venv to ensure that an activation of this venv will also activate
         the parent venv. The site directories of the parent venv should appear later in sys.path than the ones of this venv.
         """
-        sys_path_as_python_strings = ['"' + p.replace('"', '\\"') + '"' for p in list(sys.path)]
-        site_package_dir_as_python_string = '"' + self.site_packages_dir.replace('"', '\\"') + '"'
+        sys_path_as_python_strings = ['"' + p.replace('"', r"\"") + '"' for p in list(sys.path)]
```

Comment:
Do you know why this works? I would expect this to not treat the backslash as an escape because it's an r-string, yet it does.

Reply:
Snippet from the Python documentation:
```
Even in a raw literal, quotes can be escaped with a backslash, but the backslash remains in the result; for example, r"\"" is a valid string literal consisting of two characters: a backslash and a double quote; r"\" is not a valid string literal (even a raw string cannot end in an odd number of backslashes). Specifically, a raw literal cannot end in a single backslash (since the backslash would escape the following quote character). Note also that a single backslash followed by a newline is interpreted as those two characters as part of the literal, not as a line continuation.
```

Reply:
Interesting, thanks. Now I'm wondering how you would construct the `/` with an r-string then. It sounds like that would be impossible.